ccdproc/combiner.py

```diff
@@ -124,14 +124,15 @@ def weights(self):
         return self._weights
 
     @weights.setter
-    def weights(self, value):
+    def weights(self, value, axis=0):
```

Comment:
This probably won't work as expected. The `property.setter` doesn't allow passing in multiple arguments. How should this be "invoked"?

We also need a test that covers non-default axis, just to make sure it will continue to work in the future.

Reply:
@MSeifert04 I will give this some thought